appium-dotnet-driver/Appium/Enums/ClipboardContentType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenQA.Selenium.Appium.Enums
+{
+    public enum ClipboardContentType
+    {
+        PlainText,
+        Image,
```

Comment:
Is setting of images supported?

Reply:
Only for IOS. See the commits made in the integration_test project